tests/test_cliPluginLoader.py

```diff
@@ -79,6 +79,9 @@ class Suite(unittest.TestCase):
     def setUp(self):
         butler.cli.commands = None
 
+    def tearDown(self):
+        butler.cli.commands = None
```

Comment:
I'm hoping that @n8pease fixes this properly at some point but I needed to fix it here otherwise I can't run tests on my local machine (the number of cores I have means that the test partitioning was triggering a problem whereby the left over state not being reset meant that real tests in command lines failed later. It's probably right that the tearDown is the one that makes the difference.